pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -442,7 +442,7 @@ func (configgen *ConfigGeneratorImpl) buildSidecarInboundHTTPListenerOptsForPort
 		}
 	}
 
-	if features.HTTP10 || node.Metadata.HTTP10 == "1" {
+	if node.Metadata.HTTP10 == "1" {
```

Comment:
if we are removing this shouldn't it be to enable it by default? otherwise we break anyone depending on http1. 0.

Reply:
I thought people are not using this. If we want to default to true, we have to cleanup from Metadata. Also Envoy suggests setting `DefaultHostForHttp_10` https://github.com/envoyproxy/envoy/blob/5bdcdd63b348829b5e0ea1612c97168c8ce41750/generated_api_shadow/envoy/config/core/v3/protocol.proto#L150 when accept_http10 is set. So we need to think about this more.

Let me revert this change.

Reply:
BTW, Do you know the history behind controlling via feature flag? why was n't it enabled by default? Any side affects if we do that?

Reply:
I think it was just added and we didn't test it well enough (and still don't). So maybe it will have side effects we don't know about or doesn't even work